client/src/components/directory/subComponents/DirectoryTable.jsx

```diff
@@ -92,6 +95,13 @@ class DirectoryTable extends Component {
                         })
                       }
                     </TableCell>
+                    <TableCell>
+                      {
+                        scheduledCandidateIds && scheduledCandidateIds.includes(row.id)
+                          ? <Typography>{`${scheduledCandidateIds.filter(id => id === row.id).length} scheduled interview${scheduledCandidateIds.filter(id => id === row.id).length > 1 ? 's' : ''} in progress`}</Typography>
```

Comment:
do we want to see the actual number of interviews scheduled or just a simple indicator?